pkg/parsers/kernel/kernel.go

```diff
@@ -45,6 +47,26 @@ func CompareKernelVersion(a, b VersionInfo) int {
 	return 0
 }
 
+// CompareKernelVersionNproc checks if current is 4.10+ to check nproc feature
+func CompareKernelVersionNproc(a VersionInfo) int {
```

Comment:
I think return type `bool` is better than int. since you only return 1 and 0. WDYT?